examples/research_projects/quantization-qdqbert/quant_trainer.py

```diff
@@ -269,7 +269,7 @@ def set_quantizer(name, mod, quantizer, k, v):
         assert hasattr(quantizer_mod, k)
         setattr(quantizer_mod, k, v)
     else:
-        logger.warn(f"{name} has no {quantizer}")
+        logger.warning(f"{name} has no {quantizer}")
```

Comment:
Note that we don't actively maintain those ;-)

Reply:
Sure! Just covering all grounds!